### PR TITLE
[dev-v2.6] Backport CIS issue fixes to v2.6

### PR DESCRIPTION
**Pull request by rayandas**
This PR updates the security scan version to latest to bring in couple of issue fixes.
Related issues:
1. https://github.com/rancher/rancher/issues/39367
2. https://github.com/rancher/cis-operator/issues/170
3. https://github.com/rancher/cis-operator/issues/168
4. https://github.com/rancher/rancher/issues/39061
5. https://github.com/rancher/rancher/issues/39230

### CODE REVIEW

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ image:
     tag: v1.0.9
```
Comment (prachidamle) [inline packages/rancher-cis-benchmark/charts/values.yaml:8]: Seems like we need to update cis-operator tag also for backporting https://github.com/rancher/rancher/issues/38664

Comment (rayandas) [reply packages/rancher-cis-benchmark/charts/values.yaml]: Ah right! Updated the PR with the latest cis-benchmark rc tag.

File: packages/rancher-cis-benchmark/charts/values.yaml [hunk 2]
```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.9
+    tag: v1.0.10-rc2
   securityScan:
     repository: rancher/security-scan
-    tag: v0.2.8
+    tag: v0.2.9-rc6
```
Comment (prachidamle) [inline packages/rancher-cis-benchmark/charts/values.yaml]: @rayandas we should bump the images for rancher/cis-operator and  rancher/security-scan to non-rc versions now that 2.7.0 is out

Comment (rayandas) [reply packages/rancher-cis-benchmark/charts/values.yaml]: Ah! my bad. Bumped the versions.

**Review by prachidamle — APPROVED** [review]
lgtm

**Pull request merged**